src/Controls/src/Core/Platform/Windows/CollectionView/LoopableCollectionView.cs

```diff
@@ -22,6 +22,7 @@ internal class LoopableCollectionView : ICollectionView
 		public LoopableCollectionView(ICollectionView @internal)
 		{
 			_internal = @internal;
+			_internal.VectorChanged += OnItemsVectorChanged;
```

Comment:
Do we need to unsubscribe from this? Will the internal `ICollectionView` survive the lifetime of this class?